x-pack/plugins/observability_solution/apm/public/components/routing/templates/apm_main_template.tsx

```diff
@@ -118,21 +119,52 @@ export function ApmMainTemplate({
     isServerless: config?.serverlessOnboarding,
   });
 
+  let screenDescription = '';
+
+  if (!hasApmData && !hasApmIntegrations) {
```

Comment:
Made this change as the LLM was giving a suboptimal answer previously. 

When `hasApmIntegrations` would be false it would report that there might not be data because of the time range that was selected _or_ the APM integration was not installed. With this adjustment it gives a better answer.

Reply:
good one, saw this as well